src/main/java/net/greatstart/services/UserService.java

```diff
@@ -41,4 +41,7 @@ public User getUserById(long id) {
         return userDao.getAll();
     }
 
+    public User getByUsername(String name) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "name". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)